src/transformers/modeling_flax_utils.py

```diff
@@ -639,6 +640,10 @@ def from_pretrained(
                 if from_pt and os.path.isfile(os.path.join(pretrained_model_name_or_path, WEIGHTS_NAME)):
                     # Load from a PyTorch checkpoint
                     archive_file = os.path.join(pretrained_model_name_or_path, WEIGHTS_NAME)
+                elif from_pt and os.path.isfile(os.path.join(pretrained_model_name_or_path, WEIGHTS_INDEX_NAME)):
```

Comment:
BTW this will only work if the `WEIGHTS_INDEX_NAME` file is locally present, and does not include the hub.

Reply:
Yeah, let's just finalize yours. What's left to do?

Reply:
Maybe just fixing the tests, and making sure that the tests are actually good. Should be quiet straightforward☺️🙌